### ISSUE TITLE
Does not work with 1.12 MC

### ISSUE THREAD

**Issue by cybermancerr** [walrusone/HeadsLeaderBoards#1]
This plugin was working up to 1.11.2, but now does not work with 1.12 Spigot.  Please fix. I will even pay if you fix it.

**Comment by walrusone** [comment]
Yeah, I believe some of the older bukkit methods were removed in the latest spigot update. I'm pretty busy with work right now, but I'll try to get back to these as soon as I can.
